packages/trip-form/src/DateTimeSelector/index.tsx

```diff
@@ -89,9 +89,13 @@ interface DepartArriveOption {
  * @param {*} type One of the HTML5 input types.
  */
 function isInputTypeSupported(type: string): boolean {
-  const input = document.createElement("input");
-  input.setAttribute("type", type);
-  return input.type === type;
+  let retVal = false;
+  if (typeof document !== "undefined") {
```

Comment:
@fpurcell Could you also please insert a comment explaining that the check is for server-side rendering.